disco_aws_automation/disco_elastigroup.py

```diff
@@ -504,6 +504,35 @@ def update_elb(self, elb_names, hostclass=None, group_name=None):
 
         self._spotinst_call(path='/' + existing_group['id'], data=group_config, method='put')
 
+        instance_ids = {instance['instanceId']
+                        for instance in self._get_group_instances(existing_group['id'])
+                        if instance['instanceId']}
+
+        # Spotinst requires us to roll (recreate) the instances for them to pick up the new ELBs
+        # this is done in a blue/green way so doesn't cause downtime
+        self._roll_group(existing_group['id'], 100, 600)
+
+        # there isn't a roll status API so we will check progress of the operation by monitoring
+        # the instance Ids that belong to the group. Once none of the old instances are attached
+        # to the group then we know we are done
+        # give the group 10 minutes for the new instances to become healthy and the old instances to die
+        stop_time = time.time() + 600
+        while time.time() < stop_time:
+            new_instance_ids = {instance['instanceId']
+                                for instance in self._get_group_instances(existing_group['id'])
+                                if instance['instanceId']}
+
+            remaining_instances = instance_ids.intersection(new_instance_ids)
+
+            if not remaining_instances:
```

Comment:
would it make sense to also check that the same number of new instances have been created and added to the ELB?